examples/custom_distribution.ipynb

```diff
@@ -0,0 +1,425 @@
+{
```

Comment:
<p>With https://github.com/pymc-devs/pymc3/pull/4867 we're adding a mechanism for setting initial values. The code example doesn't need to include that, but maybe it should be mentioned somewhere.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/185/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>Is this guide supposed to be for 3.x or for 4.x as of now?</p>

Reply:
I believe I'm still working with 3.x